doc/docs/Python_User_Interface.md

```diff
@@ -3561,7 +3560,7 @@ Rather than writing a brand-new step function every time something a bit differe
 <a id="combine_step_funcs"></a>
 
 ```python
-def combine_step_funcs(*step_funcs): 
+def combine_step_funcs():
```

Comment:
`*step_funcs` is the only input parameter to `combine_step_funcs` and should not be removed from the documentation.